app/tasks.py

```diff
@@ -99,6 +130,29 @@ def execute(impl_url, impl_data, impl_language, transpiled_qasm, input_params, t
     # ibmq_handler.delete_token()
 
 
+def get_qpu(credentials, qpu):
```

Comment:
a similar method already exists in ibmq_handler.py please merge the functionalities of both to one single method if possible